samples/VestingTerms.example2.ocf.json

```diff
@@ -0,0 +1,68 @@
+{
+  "file_type": "OCF_VESTING_TERMS_FILE",
+  "items": [
+    {
+      "id": "all-or-nothing-with-expiration",
+      "object_type": "VESTING_TERMS",
+      "name": "Documentation: Example 2",
+      "description": "100% of the options vest on a security-specific date, within time boundaries",
+      "allocation_type": "CUMULATIVE_ROUND_DOWN",
+      "vesting_conditions": [
+        {
+          "id": "vesting-start",
+          "description": "The date on which the vesting period begins",
+          "trigger": {
+            "type": "VESTING_START_DATE"
+          },
+          "quantity": "0",
+          "next_condition_ids": [
+            "relative-expiration",
+            "absolute-expiration",
+            "qualifying-sale"
+          ]
+        },
+        {
+          "id": "relative-expiration",
+          "description": "0% vesting three years after vesting start",
+          "portion": {
+            "numerator": "0",
+            "denominator": "1"
+          },
+          "trigger": {
+            "type": "VESTING_SCHEDULE_RELATIVE",
+            "period": {
+              "length": 36,
+              "type": "MONTHS",
+              "occurrences": 1,
+              "day_of_month": "VESTING_START_DAY_OR_LAST_DAY_OF_MONTH"
+            },
+            "relative_to_condition_id": "vesting-start"
+          },
+          "next_condition_ids": []
+        },
+        {
+          "id": "absolute-expiration",
+          "description": "0% vesting after 1 Jan, 2025",
+          "quantity": "0",
+          "trigger": {
+            "type": "VESTING_SCHEDULE_ABSOLUTE",
+            "date": "2025-01-01"
+          },
+          "next_condition_ids": []
+        },
+        {
+          "id": "qualifying-sale",
+          "description": "Company is acquired for > $100MM",
+          "portion": {
+            "numerator": "1",
+            "denominator": "1"
+          },
+          "trigger": {
+            "type": "VESTING_EVENT"
+          },
+          "next_condition_ids": []
+        }
+      ]
+    }
+  ]
+}
```

Comment:
These example files are referenced from the documentation, but are included here in samples so that the validator checks them.